test/pdo_sqlsrv/pdo_azure_ad_authentication.phpt

```diff
@@ -20,15 +20,15 @@ catch( PDOException $e )
     echo "\n";
 }
 
-$stmt = $conn->query( "SELECT name FROM master.dbo.sysdatabases" );
+$stmt = $conn->query( "SELECT count(*) FROM cd_info" );
```

Comment:
this meant to test if connection actually works by using a simple query, but if you like I can change to fetching texts